tests/test_point_cloud_format.py

```diff
@@ -0,0 +1,412 @@
+import os.path as osp
+from functools import partial
+from unittest import TestCase
+
+from datumaro.components.extractor import (DatasetItem,
+                                           AnnotationType, Cuboid3D,
+                                           LabelCategories,
+                                           )
+from datumaro.components.project import Dataset
+from datumaro.plugins.pointcloud_format.converter import PointCloudConverter
+from datumaro.plugins.pointcloud_format.extractor import PointCloudImporter
+from datumaro.util.test_utils import (TestDir, compare_datasets_3d,
+                                      test_save_and_load)
+
+DUMMY_PCD_DATASET_DIR = osp.join(osp.dirname(__file__), 'assets', 'pointcloud_dataset')
+
+
+class PointCloudImporterTest(TestCase):
+    def test_can_detect_image(self):
+        self.assertTrue(PointCloudImporter.detect(DUMMY_PCD_DATASET_DIR))
+
+    def test_can_load_pcd(self):
+        pcd1 = osp.abspath(osp.join(DUMMY_PCD_DATASET_DIR, r"ds0/pointcloud/frame.pcd"))
+        pcd2 = osp.abspath(osp.join(DUMMY_PCD_DATASET_DIR, 'assets', r"ds0/pointcloud/kitti_0000000001.pcd"))
+
+        image1 = osp.abspath(osp.join(DUMMY_PCD_DATASET_DIR, r"ds0/related_images/kitti_0000000001_pcd/0000000000.png"))
+        image2 = osp.abspath(osp.join(DUMMY_PCD_DATASET_DIR, r"ds0/related_images/frame_pcd/0000000002.png"))
+
+        expected_dataset = Dataset.from_iterable([
+            DatasetItem(id='frame_000000',
+                        subset="key_id_map",
+                        annotations=[Cuboid3D(id=215,
+                                              attributes={"label_id": 0},
+                                              group=0,
+                                              points=[320.59797486700717, 979.4819186479393, 1.0372542782967524, 0.0,
```

Comment:
Would it make sense to round fp values to 2 decimal digits, like we do it for other coordinates?

Reply:
If we roundoff the coordinates will not be accurate, hence leaving as such.

Reply:
Which precision is enough? For example, we leave only 2 digits for image coordinates, because higher precision doesn't make sense in most, if not all, cases.

Reply:
@zhiltsov-max :We tested by rounding off to 2 decimals and it worked fine, modified code accordingly.